packages/datadog-plugin-winston/test/index.spec.js

```diff
@@ -203,14 +204,42 @@ describe('Plugin', () => {
               const index = semver.intersects(version, '>=3') ? 0 : 2
               const record = log.firstCall.args[index]
 
-              expect(record).to.be.an('error')
+              assert(record instanceof Error)
```

Comment:
Why not

```suggestion
              expect(record).to.be.an.instanceof(Error)
```

Reply:
That should also work. I try to just prevent using chai in general. It often does not work the way I would expect things to work. Using plain checks always work.

Reply:
I agree our tests are currently a mess. But isn't it less of a mess if we at least keep to using the same assertion library in the same file, or at least within the same test?

Reply:
It is a bit messy, I agree. It was more the way of introducing the assert library more. I think we should make more use of it instead of chai.